examples/se_2_3_localization.cpp

```diff
@@ -0,0 +1,302 @@
+/**
+ * \file se_2_3_localization.cpp
+ *
+ *  Created on: Aug 03, 2020
+ *     \author: prashanthr05
+ *
+ *  ---------------------------------------------------------
+ *  This file is:
+ *  (c) 2018 Joan Sola @ IRI-CSIC, Barcelona, Catalonia
+ *
+ *  This file is part of `manif`, a C++ template-only library
+ *  for Lie theory targeted at estimation for robotics.
+ *  Manif is:
+ *  (c) 2018 Jeremie Deray @ IRI-UPC, Barcelona
+ *  ---------------------------------------------------------
+ *
+ *  ---------------------------------------------------------
+ *  Demonstration example:
+ *
+ *  3D Robot localization and linear velocity estimation based on fixed beacons.
+ *
+ *  ---------------------------------------------------------
+ *
+ *  We consider a robot in 3D space surrounded by a small
+ *  number of punctual landmarks or _beacons_.
+ *  The robot receives control actions in the form of axial
+ *  and angular velocities, and is able to measure the location
+ *  of the beacons w.r.t its own reference frame.
```

Comment:
This doc should be updated

Reply:
Fixed in https://github.com/artivis/manif/pull/154/commits/945c4728639519d7eaa7bb47e7de7eacb9171fab